x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -9,7 +9,7 @@ export function createJestConfig({ kibanaDirectory, xPackKibanaDirectory }) {
   return {
     rootDir: xPackKibanaDirectory,
     roots: ['<rootDir>/plugins', '<rootDir>/legacy/plugins', '<rootDir>/legacy/server'],
-    moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
+    moduleFileExtensions: ['js', 'json', 'ts', 'tsx', 'node'],
```

Comment:
I updated this to be consistent with `src/dev/jest/config.js`, this change isn't needed at this time.